contingency/contingency-api/src/test/resources/identifierContingencyList.json

```diff
@@ -2,7 +2,6 @@
   "type" : "identifier",
   "version" : "1.0",
```

Comment:
Just a question for me: do we have versioning for contingency list serializations?

Reply:
yes we have

Reply:
Thanks, it should be handled in this PR then. I'm leaving a comment :)

Reply:
well I think it is not used
```java
public interface ContingencyList {

    // VERSION = 1.0 : first version
    String VERSION = "1.0";

    static String getVersion() {
        return VERSION;
    }
}
```

Reply:
It might be updated to `1.1` (or at least override in `IdentityContingencyList`) and handled in deserializer

Reply:
In this case we can have in the serializer:
```jsonGenerator.writeStringField("version", identifierContingencyList.getVersion());```

Reply:
to conclude we add a version in IdentifierContingencyList and we upgrade to 1.1 ? and add serialization for version and check on the field we delete in this pr ?

Reply:
Yes